src/style.css

```diff
@@ -0,0 +1,132 @@
+:root {
+  --main-color: #060b35;
+  --secondary-color: #616070;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+p {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  color: var(--main-color);
```

Comment:
how do you think - puting major color on html tag is comfortably? i don't think so. maybe it will much better if you put unice color parameter to unice block